src/Event/Productions/Tuple.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\Productions;
+
+class Tuple
```

Comment:
Should we call this `EventPair`? 
`Tuple` is a structural name, similar to `String` or `Array`. You wouldn't call a VO `string` without giving it more domain meaning, so I wouldn't use `tuple` either.